### ISSUE TITLE
Update error

### ISSUE THREAD

**Issue by angristan** [FreshRSS/FreshRSS#1516]
I get this error when upgrading from 1.6.2 to 1.6.3 : 

```
2017/05/01 16:21:38 [error] 20736#20736: *11711 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught Error: Call to undefined function need_info_update() in /srv/freshrss/app/Controllers/updateController.php:196
Stack trace:
#0 /srv/freshrss/lib/Minz/Dispatcher.php(119): FreshRSS_update_Controller->applyAction()
#1 /srv/freshrss/lib/Minz/Dispatcher.php(48): Minz_Dispatcher->launchAction('applyAction')
#2 /srv/freshrss/lib/Minz/FrontController.php(84): Minz_Dispatcher->run()
#3 /srv/freshrss/p/i/index.php(46): Minz_FrontController->run()
#4 {main}
  thrown in /srv/freshrss/app/Controllers/updateController.php on line 196" while reading response header from upstream, client: 62.210.217.5, server: rss.angristan.xyz, request: "GET /p/i/?c=update&a=apply HTTP/2.0", upstream: "fastcgi://unix:/run/php/php7.0-fpm.sock:", host: "rss.angristan.xyz"

```

It has been working for months, did I break something ?

**Comment by Alkarex** [comment]
Hello,
There is a bug in 1.6.2 for the automatic update via ZIP. See https://github.com/FreshRSS/FreshRSS/issues/1488#issuecomment-293998667

**Comment by angristan** [comment]
Oh, ok ! Should I use the workaround ?

**Comment by Alkarex** [comment]
Yes. The proper fix will first be available in 1.7.0 (not released yet), so you will have to do the same from 1.6.3 to 1.7.0.
